index.html

```diff
@@ -386,7 +386,7 @@ <h2 class="lead secondary-heading"><a href="/packages/">Packages</a></h2>
 
       <br>
 
-      <p>Julia has been downloaded over 40 million times and the Julia community has registered <a href="https://juliahub.com/ui/Packages">over 9,100 Julia packages</a> for community use.
+      <p>Julia has been downloaded over 45 million times and the Julia community has registered <a href="https://juliahub.com/ui/Packages">over 9,500 Julia packages</a> for community use.
```

Comment:
I don't like such non-specific text. I'm willing and able to make PRs regularly, e.g. next for "over 10,000 package" milestone ("thousands" is worse than "10,000", though it could mean 50,000 or just 3,000...), but I'm open to some automated solution to provide numbers, and may look into that.

Reply:
I do think the specific numbers are valuable and updating them twice a year is ok.